packages/styles/scss/components/tableofcontents/README.md

```diff
@@ -0,0 +1,24 @@
+### SCSS
+
+#### Usage
+
+Import the package css into the top of your main CSS file.
+
+```css
+@import '@carbon/ibmdotcom-styles/scss/patterns/tableofcontents/index';
```

Comment:
@sannminwin should update the reference to `components` instead of `patterns` folder

Reply:
updated.